docker/nginx/nginx.conf

```diff
@@ -38,6 +38,12 @@ http {
     map $host $content_security_policy {
         default "default-src 'self'; connect-src 'self' https://supremum.gewis.nl/ https://analytics.gewis.nl/ https://www.googleapis.com/calendar/; script-src 'self' https://analytics.gewis.nl/ 'unsafe-eval' 'unsafe-inline'; img-src 'self' https://*.gewis.nl/ data: https://images.gewis.nl/ https://images.test.gewis.nl/; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com/; font-src 'self' https://fonts.gstatic.com/; frame-src 'self' https://analytics.gewis.nl/ https://www.youtube-nocookie.com/embed/ https://player.vimeo.com/video/; object-src 'none'; frame-ancestors 'self'; form-action 'self'";
     }
+    map $host $referrer_policy {
+        default "strict-origin-when-cross-origin";
+    }
+    map $host $permissions_policy {
+        default "accelerometer=("https://www.youtube-nocookie.com" "https://player.vimeo.com"), ambient-light-sensor=(), autoplay=(), battery=(), camera=(), cross-origin-isolated=(), display-capture=(), document-domain=(), encrypted-media=("https://www.youtube-nocookie.com" "https://player.vimeo.com"), execution-while-not-rendered=(), execution-while-out-of-viewport=(), fullscreen=(self "https://www.youtube-nocookie.com" "https://player.vimeo.com"), geolocation=(), gyroscope=("https://www.youtube-nocookie.com" "https://player.vimeo.com"), keyboard-map=(), magnetometer=(), microphone=(), midi=(), navigation-override=(), payment=(), picture-in-picture=("https://www.youtube-nocookie.com" "https://player.vimeo.com"), publickey-credentials-get=(), screen-wake-lock=(), sync-xhr=(), usb=(), web-share=(), xr-spatial-tracking=(), clipboard-write=(self), interest-cohort=()";
+    }
 
```

Comment:
NGINX does not accept this due to quotes in the `Permissions-Policy`. The outer qoutes should be changed to `'`, as `"` is required for the URLs.

Reply:
I fixed and tested it now